challenge-1/bmi-comparator.js

```diff
@@ -0,0 +1,18 @@
+class Person {
+
+    constructor(mass, height) {
+        this._mass = mass;
+        this._height = height;
+    }
+
+    get bodyMassIndex() {
+        return this._mass / (this._height * this._height)
+    }
+}
+
+const mark = new Person(78, 1.69);
+const john = new Person(92, 1.95);
+
+const isMarkBmiHigher = mark.bodyMassIndex > john.bodyMassIndex;
+
+console.log(isMarkBmiHigher ? mark.bodyMassIndex : john.bodyMassIndex);
```

Comment:
Oooh, nice touch with the ternary operator but a question; do we have a way to know the name that have the biggest body mass?

Reply:
In that case I would have that have other property in the class Person to store the name to the person. And instead of the ternary operator I would use a if to print the name and the bmi of the winner person.

So

```javascript
if (isMarkBmiHigher) {
    console.log(`${mark.name} is the winner with a bmi of: ${mark.bodyMassIndex}`);
} else {
    console.log(`${john.name} is the winner with a bmi of: ${john.bodyMassIndex}`);
}
```

Reply:
Hey can we implement the new solution>

Reply:
Done